pkg/appgw/internaltypes.go

```diff
@@ -12,6 +12,8 @@ package appgw
 import (
 	"crypto/md5"
 	"fmt"
+	"github.com/Azure/application-gateway-kubernetes-ingress/pkg/utils"
```

Comment:
Since this package is local to this repo - we adopted the convention to put it in the last group -- where groups of packages are separated by a new line.